docs/install_guides/installing-openvino-docker-linux.md

```diff
@@ -1,68 +1,90 @@
+You can also try our [Tutorials](https://github.com/openvinotoolkit/docker_ci/tree/master/docs/tutorials) which demonstrate the usage of Docker containers with OpenVINO. 
 
-You don't need to do specific steps to configure an Intel® Distribution of OpenVINO™ toolkit Dockerfile for CPU. You can use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles) or generate a Dockerfile with your setting via [DockerHub CI Framework](https://github.com/openvinotoolkit/docker_ci).
+### Step 2: Configure and run the image for different devices
 
-### Run the Docker* Image for CPU
+#### Using Docker image on CPU
 
-Run the image with the following command:
+Note the following things:
 
-```sh
+- Kernel reports the same information for all containers as for native application, for example, CPU, memory information.
+- All instructions that are available to host process available for process in container, including, for example, AVX2, AVX512. No restrictions.
+- Docker\* does not use virtualization or emulation. The process in Docker* is just a regular Linux process, but it is isolated from external world on kernel level. Performance loss is minor.
+
+To use the OpenVINO Docker image on CPU, you don’t need extra configurations. Run the Docker image with the following command:
+```
 docker run -it --rm <image_name>
 ```
 
-## Use a Docker* Image for GPU
-
-### Configure a Docker* Image for GPU
+#### Using Docker image on GPU
 
 > **NOTE**: Only Intel® integrated graphics are supported.
 
-**Prerequisites:**
+**Prerequisites**
 
 - GPU is not available in container by default, you must attach it to the container.
 - Kernel driver must be installed on the host.
 - Intel® OpenCL™ runtime package must be included into the container.
 - In the container, non-root user must be in the `video` and `render` groups. To add a user to the render group, follow the [Configuration Guide for the Intel® Graphics Compute Runtime for OpenCL™ on Ubuntu* 20.04](https://github.com/openvinotoolkit/docker_ci/blob/master/configure_gpu_ubuntu20.md).
 
-To configure a OpenVINO Docker* image with access to GPU, add the following commands to a Dockerfile:
+To launch a Linux image on Windows Subsystem for Linux 2, note the following things:
+
+- Only Windows 10 with 21H2 update or above installed and Windows 11 are supported.
+- Intel GPU driver on Windows host with version 30.0.100.9684 or above need be installed. Please see [this article](https://www.intel.com/content/www/us/en/artificial-intelligence/harness-the-power-of-intel-igpu-on-your-machine.html#articleparagraph_983312434) for more details.
+- The Docker images for 2022.1 release contain preinstalled recommended version of OpenCL Runtime with WSL2 support.
+
+##### Step 1: Configure the image for GPU
```

Comment:
We can make this step optional and provide instructions for users who want to get the image with the specified custom version of OpenCL runtime installed. Something like this:
> By default, OpenVINO Docker* images are distributed with installed recommended version of the Intel® Graphics Compute Runtime for oneAPI Level Zero and OpenCL™ Driver for the operating system in the image. If you want to build an image with a custom version of OpenCL™ runtime included, you need to modify the dockerfile using the lines below (the 19.41.14441 version is used as an example) and build the image manually.

Reply:
Does it mean the available dockerfiles on Github  contains recommended OpenCL version too?

Reply:
Yes, prebuilt Docker images are built from their respective dockerfiles available on GitHub. 
So, if a user builds an image from a dockerfile without making changes to it, the recommended OpenCL version will be installed in the built image as well.

Reply:
OK, thanks. Then I think this change has been included in the latest version.